app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager.rb

```diff
@@ -5,20 +5,31 @@ class ManageIQ::Providers::IbmCloud::VPC::CloudManager < ManageIQ::Providers::Cl
   require_nested :Vm
 
   include ManageIQ::Providers::IbmCloud::VPC::ManagerMixin
-
+  delegate :cloud_volumes, :to => :storage_manager
+  has_one :storage_manager,
+          :foreign_key => :parent_ems_id,
+          :class_name  => "ManageIQ::Providers::IbmCloud::VPC::StorageManager",
+          :autosave    => true,
+          :dependent   => :destroy
   before_create :ensure_managers
   before_update :ensure_managers_zone
 
   def ensure_managers
     ensure_network_manager
     ensure_managers_zone
+    ensure_storage_manager
```

Comment:
Let's move this up before `ensure_managers_zone` so that we can keep the storage_manager.zone_id in sync